apps/meteor/tests/e2e/translations.spec.ts

```diff
@@ -29,8 +29,24 @@ test.describe('Translations', () => {
         expect((await setUserPreferences(api, { language: '' })).status()).toBe(200);
     });
 
-    // Browser language not working on CI for some reason. This test passes locally.
-    test.describe.skip('Browser', async () => {
+    test('expect to keep chosen language after refresh', async ({ page, api }) => {
+        await page.goto('/home');
+        await page.waitForTimeout(5000);
```

Comment:
Do we need this fixed timeout? 🤔 Wouldn't it be possible to wait for an element instead?